src/pages/LoginModule/index.tsx

```diff
@@ -0,0 +1,26 @@
+import { useContext } from "react";
+import { UserContext } from "store/user/context";
+import { deleteAuthToken } from "utils/deleteAuthToken";
+import { LoginForm } from "components/modules/forms/LoginForm";
+
+const LoginModule = () => {
```

Comment:
This should be rebuilded, routing should check if user is logged in and then redirect user to the specific page